app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccinated_friends = 0
```

Comment:
`vaccinated_friends` variable is not used other than for incrementation, there is no logic dependant on it. Remove this variable